packages/tempus-client_v3/src/components/Markets/MarketsPools/MarketsPools.tsx

```diff
@@ -44,7 +45,7 @@ const MarketsPools = (): JSX.Element => {
   const onShowMoreClick = useCallback((chain: Chain) => {
     setVisibleChainPools(prevState => ({
       ...prevState,
-      [chain]: (prevState[chain] || NUMBER_OF_CARDS_PER_PAGE) + NUMBER_OF_CARDS_PER_PAGE,
+      [chain]: (prevState[chain] || NUMBER_OF_CARDS_PER_PAGE) + 2 * NUMBER_OF_CARDS_PER_PAGE,
```

Comment:
Since you've used `2 * NUMBER_OF_CARDS_PER_PAGE` in a few places, why don't you change `NUMBER_OF_CARDS_PER_PAGE` to 6 and introduce `const NUMBER_OF_INITIAL_CARDS = 3`?

```suggestion
      [chain]: (prevState[chain] || NUMBER_OF_INITIAL_CARDS) + NUMBER_OF_CARDS_PER_PAGE,
```

IMO it's clearer this way instead of figuring out why we used `2 *`.

Reply:
fixed